src/components/charts/continue-chart.tsx

```diff
@@ -5,23 +5,10 @@ import moment from 'moment';
 import { Bar } from 'react-chartjs-2';
 import { TimelineStat } from '../../types/stats';
 import { ContributeType } from '../../types/contribute';
+import { useTranslation } from '../../server/i18n';
 
 moment.locale('is');
 
-const dateFormatter = (item: any) => {
-    const today = moment().diff(item, 'days');
-    if (today == 0) {
-        return 'í dag';
-    } else if (today == 5 || today == 3) {
-        return '';
-    } else {
-        if (today == 6) {
-            return 'fyrir viku';
-        }
-        return '';
-    }
-};
-
```

Comment:
I had to move this function into the component to access to the translation strings